src/scss/initial/_root.scss

```diff
@@ -16,6 +16,6 @@ html {
     0 100%
   );
 
-  font-size: config.size('rem');
+  font-size: calc(1.25em + 0.5vw);
```

Comment:
@mirisuzanne Feel free to adjust this if you wanted it stored in a custom property still.